doc/nvim-tree-lua.txt

```diff
@@ -317,6 +317,7 @@ applying configuration.
       sort = {
         sorter = "name",
         folders_first = true,
+        files_first = false,
```

Comment:
Sorry for not checking myself, but what will happen when user sets both `folders_first` and `files_first` to `true`?

Maybe some extra sorters, perhaps configurable, should be provided instead of adding specialised options?

Reply:
Tested OK